Source/DafnyPipeline.Test/Issue1355.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Diagnostics.Contracts;
+using System.IO;
+using Bpl = Microsoft.Boogie;
+using BplParser = Microsoft.Boogie.Parser;
+using Microsoft.Dafny;
+using Xunit;
+
+namespace DafnyPipeline.Test {
+  public class Issue1355 {
+    [Fact]
+    public void Test() {
+      ErrorReporter reporter = new ConsoleErrorReporter();
+      var options = new DafnyOptions(reporter);
+      options.DafnyPrelude = "../../../../../Binaries/DafnyPrelude.bpl";
+      DafnyOptions.Install(options);
+
+      var programString = @"trait Trait<A, B> { }";
```

Comment:
It might be nice if we have a Dafny program that exercises all of the Dafny languages features, which we could use for both a Dafny and a Boogie print test.

Reply:
I agree. But let's do that in a different PR.

Reply:
OK